assets/helpers/internationalisation/country.js

```diff
@@ -398,20 +391,24 @@ function setCountry(country: IsoCountry) {
 }
 
 function handleCountryForCountryGroup(
-  targetCountryGroup: 'International' | 'EURCountries' | 'NZDCountries',
+  targetCountryGroup: 'International' | 'EURCountries' | 'NZDCountries' | 'GBPCountries' | 'AUDCountries',
```

Comment:
Is it worth breaking this out into a declared type, and then applying type annotations to the `paths` and `defaultCountry` objects below?